app/controllers/submit.js

```diff
@@ -632,24 +630,19 @@ export default Ember.Controller.extend(Analytics, BasicsValidations, NodeActions
                     label: 'Submit - Save and Continue, Existing Node Existing File'
                 });
 
-            const node = this.get('node');
-            const currentTitle = node.get('title');
-            const nodeTitle = this.get('nodeTitle');
+            const model = this.get('model');
+            const currentTitle = model.get('title');
+            const title = this.get('title');
 
             this.set('basicsAbstract', this.get('node.description') || null);
 
             return Promise.resolve()
                 .then(() => {
-                    if (currentTitle === nodeTitle) {
-                        return;
-                    }
-
-                    node.set('title', nodeTitle);
-                    return node.save();
+                    if (currentTitle !== title)
```

Comment:
I believe our eslint curly setting is set to multiline: https://eslint.org/docs/rules/curly#multi-line
So this will fail without the braces once eslint is enabled.